activesupport/lib/active_support/callbacks.rb

```diff
@@ -782,7 +782,7 @@ def deprecated_false_terminator # :nodoc:
 
       def display_deprecation_warning_for_false_terminator
         ActiveSupport::Deprecation.warn(<<-MSG.squish)
-          Returning `false` in Active Record and Active Model callbacks will not implicitly halt a callback chain in the next release of Rails.
+          Returning `false` in Active Record and Active Model callbacks will not implicitly halt a callback chain in Raiils 5.1.
```

Comment:
I assumed since the commit that introduced the deprecation first appeared in v5.0.0.beta1, that this would be deprecated in v5.1.0 😬 

cc @claudiob